src/scripts/main.js

```diff
@@ -1,3 +1,9 @@
 'use strict';
 
-// write your code here
+const rows = document.querySelectorAll('tr');
+
+rows.forEach(row => {
+  const cell = row.cells[1].cloneNode(true);
+
+  row.insertBefore(cell, row.children[4]);
```

Comment:
Instead of using the hardcoded index 4, you can use row.cells.length - 1 to get the index of the last column.